packages/markers/src/browser/problem/problem-frontend-module.ts

```diff
@@ -0,0 +1,30 @@
+/*
+* Copyright (C) 2017 TypeFox and others.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+import { ContainerModule } from "inversify";
+import { ProblemWidget } from './problem-widget';
+import { ProblemContribution } from './problem-contribution';
+import { MarkerManager } from '../marker-manager';
+import { createProblemWidget } from './problem-container';
+import { CommandContribution, MenuContribution, KeybindingContribution } from "@theia/core/lib/common";
+
+import '../../../src/browser/style/index.css';
```

Comment:
This looks weird to me ?

Reply:
what part? we are in lib and want to load something from src.

Reply:
The importing from .css  why do we need to import the style there?

Reply:
I guess I would need to check the webpack loader..

Reply:
You need to require it explicitly. Otherwise, webpack does not know that it has to be bundled too. Another alternative would be to "manually" copy the CSSs (just like any other static assets) with the webpack copy plug-in. At least, that is my understanding.

Reply:
Ha OK thx :)